keyboards/takashicompany/spreadwriter/readme.md

```diff
@@ -0,0 +1,35 @@
+# takashicompany/spreadwriter
+
+![takashicompany/spreadwriter](https://dl.dropboxusercontent.com/s/c26r9bugx9xw5df/01.jpg)
```

Comment:
Pleas rehost this on imgur.

Reply:
Fixed. https://github.com/qmk/qmk_firmware/pull/17031/commits/d71ace8e155e66bdbc6c33b2cde28c53f9fccd82